pages/api/proposal/[proposalId]/votes.ts

```diff
@@ -0,0 +1,45 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { z } from 'zod';
+import { subgraphService } from '../../../../lib/services/subgraph.service';
+import { restrictHandlerMethods } from '../../../../lib/util/api';
+import { buildVotesWithLikes } from '../../../../lib';
+
+const QuerySchema = z.object({
+  proposalId: z.string(),
+  page: z.number().optional(),
+  limit: z.number().optional(),
+  order: z.enum(['desc', 'asc']).default('desc'),
+});
+
+type Query = z.infer<typeof QuerySchema>;
+async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
thanks for this! `proposals/<id>/votes` would be more resource oriented, nbd though https://cloud.google.com/compute/docs/naming-resources